backend/timesheets/models.py

```diff
@@ -151,37 +155,43 @@ class Timesheet(db.Model):
     start_work_time = db.Column(db.String(20), nullable=True)
     end_work_time = db.Column(db.String(20), nullable=True)
     hours_worked = db.Column(db.Integer, nullable=False)
-    status = db.Column(db.String(20), nullable=False)
+    status = db.Column(db.Boolean, default= False, nullable=False) #changed to boolean from string
```

Comment:
Keep this as string, we will be using three states: Pending, Approved and Disapproved